deploy/mayastor-daemonset.yaml

```diff
@@ -60,6 +61,8 @@ spec:
           mountPath: /var/local/mayastor/
         - name: config
           mountPath: /var/local/mayastor/config.yaml
+        - name: child-status-config
```

Comment:
let us not put this stuff in the deployment files right now as it will end up in people configs when doing a release. It creates issues when troubleshooting among others. In fact, could you remove (for now) both config files? I'm fine if we create a separate file here, that does include them.

Reply:
Done - I've removed the config from here and created a separate mayastor-daemonset-config.yaml file which does use them